studio-docs/source/org/graphs.mdx

```diff
@@ -44,6 +44,12 @@ The History view in Apollo Studio lets you view the timeline of changes made to
 
 **Only schema changes that you push to Studio are included in this timeline**, which is one of the most important reasons to [include schema registration in your continuous delivery pipeline](../schema/cli-registration/#registering-with-continuous-delivery).
 
+## Setting graph visibility (Enterprise only)
+
+By default, deployed graphs are visible to all members of your organization. If you have an Enterprise plan, you can set a deployed graph to instead be visible only to members you invite. 
```

Comment:
You use the word "invite" twice in this section but it's not a word we use to describe adding users to graphs in the UI or on the members.mdx page.